addons/partner_autocomplete/static/tests/partner_autocomplete_tests.js

```diff
@@ -346,4 +347,15 @@ QUnit.module('partner_autocomplete', {
             assert.strictEqual(target.querySelector(`[name=${fieldName}] input`).value, expectedValue, `${fieldName} should be filled`);
         }
     });
+
+    QUnit.test("Show confirmation dialog on input blur", async function (assert) {
+        assert.expect(1);
+        await makeView(makeViewParams);
+        const input = target.querySelector("[name=parent_id] input.o-autocomplete--input.o_input");
+        await triggerEvent(input, null, "focus");
+        await click(input);
+        await editInput(input, null, "go");
+        await triggerEvent(input, null, "blur");
+        assert.containsOnce(target, ".o_technical_modal");
```

Comment:
Hi @aab-odoo. In this forward port, the `.modal-open` is not longer available. Is it okay if I use `.o_technical_modal` to check the presence of the dialog?

Reply:
Hello @caburj 

Sure, or even simply .o_dialog

Reply:
I used it and it works.